doc/ale-ruby.txt

```diff
@@ -14,6 +14,25 @@ g:ale_ruby_brakeman_options                       *g:ale_ruby_brakeman_options*
 
 
 ===============================================================================
+rails_best_practices                            *ale-ruby-rails_best_practices*
+
+g:ale_ruby_rails_best_practices_executable *g:ale_ruby_rails_best_practices_executable*
```

Comment:
These lines are too long, but I'm not sure how to split them while satisfying the vim help syntax.